build/travis/deploy.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Deploy artifacts (e.g. dmg, deb files) built by Travis to downloads.mixxx.org.
+# Run within the cmake_build directory.
+
+set -eu -o pipefail
+
+USER=mixxx
+HOSTNAME=downloads-hostgator.mixxx.org
+TRAVIS_DESTDIR=public_html/downloads/builds/travis
+SSH_KEY=../build/certificates/downloads-hostgator.mixxx.org.key
+SSH="ssh -i ${SSH_KEY} -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+
+if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]; then
+  echo "Skipping deploy since we are in a pull request."
+  exit 0
+fi
```

Comment:
Can we deploy unsigned builds for PRs? This would help a lot with getting people to test PRs.

Reply:
Oh, nevermind. This script is just deploying, not signing.

Reply:
I don't think this check in the script is necessary:

> Note that pull request builds skip the deployment step altogether.

https://docs.travis-ci.com/user/deployment/#pull-requests